tsconfig.base.json

```diff
@@ -14,6 +14,12 @@
     "skipLibCheck": true,
     "jsx": "react",
     "lib": ["es6", "dom"],
-    "strictPropertyInitialization": true
+    "strictPropertyInitialization": true,
+    "paths": {
+      "xstate": ["./packages/core/src"],
+      "xstate/lib/*": ["./packages/core/src/*"],
+      "@xstate/graph": ["./packages/xstate-graph/src"],
+      "@xstate/fsm": ["./packages/xstate-fsm/src"]
+    }
```

Comment:
what's the practical outcome for this for us? given we use composite projects here this all gets correctly re-mapped already (apart from the `xstate/lib/*`)

Reply:
This is so that tests do not require build to be ran beforehand - so jest no longer depends on build artifacts, which makes it safe to ignore build artifacts in jest's watcher.